src/core/sys/posix/sys/ttycom.d

```diff
@@ -108,3 +108,113 @@ version (OSX)
     enum uint SLIPDISC = 4;       // serial IP discipline
     enum uint PPPDISC  = 5;       // PPP discipline
 }
+else version (OpenBSD)
+{
+    struct winsize {
```

Comment:
Curly braces on their own lines.

Reply:
You want me to fix the curly braces in the OSX section as well?

Reply:
No, only for the code you've added.

Reply:
fixed.